CameraBrowser/.gitignore

```diff
@@ -0,0 +1,405 @@
+# globs

```

Comment:
This file might not be needed if the same one or similar is present in the root of the project: https://github.com/xamarin/mac-samples/blob/main/.gitignore 

Try removing it and add whatever is needed there.